content/gsoc-2024/index.md

```diff
@@ -0,0 +1,110 @@
++++
+title = "Google Summer of Code 2024 Wrap-up"
+date = 2024-11-21
+[taxonomies]
+authors = ["Aaron Allen"]
+categories = ["Google Summer of Code", "Haskell.org"]
```

Comment:
Question: We used to call this "Haskell Summer of Code" sometimes. Which category should we use? HSoC? GSoC?

Reply:
I think you're referring to past years when we ran our own independent program (due to not being selected for GSoC), in which case `s/Google/Haskell`. I suppose they shouldn't be separate categories for the purpose of this blog... perhaps `Summer of Code` should be used for the category?

Reply:
You're right, `Summer of Code` looks more appropriate, since Google doesn't always select us.